src/DeSmart/Pagination/Paginator.php

```diff
@@ -149,7 +149,11 @@ public function getUrl($page) {
     $parameters[$this->factory->getPageName()] = $page;
     $absolute = (null === $this->routeConfig['absolute']) ? true : $this->routeConfig['absolute'];
 
-    return $this->urlGenerator->route($this->routeConfig['name'], $parameters, $absolute, $this->routeConfig['instance']);
+    // allow adding hash fragments to url
+    $fragment = $this->buildFragment();
+    $generatedRoute = $this->urlGenerator->route($this->routeConfig['name'], $parameters, $absolute, $this->routeConfig['instance']);
```

Comment:
Little code style thing: for local variables we're using snake case. Could you correct this?